.github/workflows/kind-test.yaml

```diff
@@ -0,0 +1,24 @@
+name: kind-test
+
+on: [push, pull_request]
+
+jobs:
+  kind-test:
+    runs-on: ubuntu-latest
+    env:
+      GO111MODULE: on
+    steps:
+      - name: Set up Go 1.14
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.14
+      - uses: actions/checkout@v2
+      - name: Build KubeEye
+        run: make ke
+      - name: Creating kind cluster
+        uses: helm/kind-action@v1.0.0-rc.1
+      - name: Run KubeEye
+        run: |
+          rm -rf temp & mkdir temp
+          mv ke temp/
+          ./temp/ke diag
```

Comment:
Why don't use `ke` directly? Make them to one line: `./ke diag`.

And maybe we need to check the exit code here. Make sure the action is failed if the exit code is not 0.

Reply:
This is to test packr2.

Reply:
> This is to test packr2.

I dont' understand this. No matter what, remove the temp dir is not related with `packr2`. If you try to clean the dir, maybe try to use `make clean ke`.

Reply:
Packr is a simple solution for bundling static assets inside of Go binaries. It needs to change the directory to test it works.